atd-toolbox/data_model/python/10_make_core_tables.py

```diff
@@ -92,7 +92,8 @@ def make_column_sql(columns, full_table_name):
             fk_cascade = "on update cascade on delete cascade"
         elif constraint and "not null default false" in constraint and "_edits" in full_table_name:
             constraint = ""
-
+        elif constraint == "not null" and column_name in ['unit_nbr', 'crash_timestamp'] and "_edits" in full_table_name:
+            constraint = ""
```

Comment:
can you refresh me on the purpose of this script/what this is doing? and is this line in particular making sure that `crash_timestamp` and `unit_nbr` can be null in the edits table?

Reply:
yes! this is the script that generates all of the `create table` migrations for crashes/units/people/charges. it does this by parsing out the `column_metadata.json` file. 

and just like you said, this `elif` block makes sure that those fields can be null in the edits table.